.flake8.ci

```diff
@@ -0,0 +1,3 @@
+[flake8]
```

Comment:
How does flake8 know to use this file instead of the other one?

Reply:
By default flake will look for a few files. `.flake8` is one such file (`tox.conf` and another one I cannot remember at the moment).

Reply:
But now that I've been working through the tests module I'm rethinking this strategy... perhaps I can change the `check_lint` function to make use of the config files better and I can then just call `python -m tests --lint-only` and still get the config file sweetness.

Reply:
I've not done this yet, but we can open a new issue if this is important to get it to the stage where automation is working.

Reply:
yeah, let's table this